src/components/TaxonomyTerms/TaxonomyTerms.js

```diff
@@ -4,6 +4,7 @@ import styles from './TaxonomyTerms.module.scss';
 
 export default function TaxonomyTerms({ post, taxonomy }) {
   const termLinks = post?.[taxonomy]?.()?.edges.map((edge, index) => {
+    // eslint-disable-next-line no-unsafe-optional-chaining
```

Comment:
There is no optional chaining being done in the next line. Do we need this?

Reply:
Sorry wrong place